src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -27,6 +23,13 @@ public FindCommand(Set<String> keywords) {
         this.keywords = keywords;
     }
 
+    private static Collection<String> lowerCaseCollection(Collection<String> strings){
```

Comment:
Instead of using lowerCaseCollection maybe you can consider adding a getLowerCaseWordsInName in the Name class and just compare it with lower cased keywords?

Reply:
+1 for the above comment